### DISCUSSION TITLE
Installing Python versions?

### DISCUSSION THREAD

**Discussion by swamper123** [python-poetry/poetry#4161] [Q&A] [answered]
Hello together,

if I understood things right, poetry is able to install different python versions for a project as well or am I missunderstanding something?
If it can, can it install python versions with "--enable-optimisations" as well? If yes, how would the command look like?

**Comment by ghost** [comment, marked as answer]
No, poetry does not install python versions. that is a requirement that you can specify in the `pyproject.toml`. if the requirement of python version is not satisfied, the installation fails. Its basically telling poetry at what minimum version required will your project run.
